posts/2024-04-04/index.qmd

```diff
@@ -0,0 +1,96 @@
+---
+title: "Basic taxonomic classification of long and short reads with Kraken2"
+author: "Diego Gomes"
+date: "2024-04-04"
+bibliography: ref.bib
+freeze: auto
+categories: [debbuging new]
+---
+
+```{=html}
+<style>
+body {
+text-align: justify}
+</style>
+```
+![Basic taxonomic classification of long and short reads with Kraken2](Fig.png)
+
+The taxonomic classification of reads from sequencing experiments has become a common task for most computational biology and bioinformatics projects. To facilitate this process, Kraken2 was developed to make it faster and computationally less costly [@wood_improved_2019].
+
+To assist those who are new to taxonomic classification activities and even metagenomics, we will describe a brief tutorial here for performing an analysis with Kraken2. It's worth noting that all commands shown here are extensively described in the [software's manual](https://github.com/DerrickWood/kraken2/blob/master/docs/MANUAL.markdown).
+
+### Kraken2 installation
+
+**Option 1**: Clone from github repository
+
+```{bash}
```

Comment:
What you think about not using {bash} and running {r, eval = FALSE} in all chunks. This way html will have a better parsing by knitr.
```suggestion
```{r, eval = FALSE}
```